docker-compose.yml

```diff
@@ -1,6 +1,11 @@
 version: '3.1'
 services:
   ae_mdw:
+    healthcheck:
+      test: ["CMD", "curl", "-f", "localhost:4000/status"]
+      interval: 1m
+      timeout: 50s
+      retries: 3
     build:
       context: .
       dockerfile: ./Dockerfile
```

Comment:
```suggestion
    image: aeternity/ae_mdw:latest
```

Reply:
I think that would make more sense (once we have proper tagging of images)

Reply:
We are close to a release, it makes sense.

Reply:
this is for deployment anyway, so it wont mess up with anything because it won't be deployed earlier

Reply:
Thanks @venimus, will do after our first release (after stable latest is available)